dir.props

```diff
@@ -89,17 +89,10 @@
   <Import Project="$(ProjectDir)dependencies.props" />
 
   <PropertyGroup>
-    <!--
-        Temporarily update RootRepo to known-good
-        repo project until all repos are building clean.
-        When all repos build, this can be changed back
-        to cli.
     <RootRepo>cli</RootRepo>
-    -->
-
-    <!-- Temporarily update RootRepo to core-setup repo project
+    <!-- Temporarily update RootRepo for Windows to core-setup repo project
```

Comment:
Maybe the comment should be split into one each for Windows vs. arm? (Also justifies having two props setting the same value IMO.)

Reply:
Added an additional comment for  arm.